Neos.ContentRepository/Classes/Package.php

```diff
@@ -55,7 +55,7 @@ public function boot(Bootstrap $bootstrap)
         $context = $bootstrap->getContext();
         if (!$context->isProduction()) {
             $dispatcher->connect(Sequence::class, 'afterInvokeStep', function ($step) use ($bootstrap) {
-                if ($step->getIdentifier() === 'typo3.flow:systemfilemonitor') {
+                if ($step->getIdentifier() === 'neos.flow:systemfilemonitor') {
                     $nodeTypeConfigurationFileMonitor = FileMonitor::createFileMonitorAtBoot('TYPO3CR_NodeTypesConfiguration', $bootstrap);
```

Comment:
Seems to be consistent throughout the package. Would rather change that in an other PR.